sites/galamtv.kz/galamtv.kz.config.js

```diff
@@ -0,0 +1,64 @@
+const axios = require('axios')
+const dayjs = require('dayjs')
+const utc = require('dayjs/plugin/utc')
+const timezone = require('dayjs/plugin/timezone')
+const customParseFormat = require('dayjs/plugin/customParseFormat')
+
+dayjs.extend(utc)
+dayjs.extend(timezone)
+dayjs.extend(customParseFormat)
+
+module.exports = {
+    site: 'galamtv.kz',
+    timezone: 'Asia/Almaty',
+    days: 2,
+    request: {
+        method: 'GET',
+        headers: {
+            Referer: 'https://galamtv.kz/',
+            Origin: 'https://galamtv.kz',
+            Accept: '*/*',
+            'Accept-Encoding': 'gzip, deflate, br, zstd',
+        }
+    },
+    url({ channel, date }) {
+        const todayEpoch = date.startOf('day').utc().valueOf()
+        const nextDayEpoch = date.add(1, 'day').startOf('day').utc().valueOf()
```

Comment:
You just need to use timestamp in seconds to get a valid url.

https://day.js.org/docs/en/display/unix-timestamp

```suggestion
        const todayEpoch = date.startOf('day').unix()
        const nextDayEpoch = date.add(1, 'day').startOf('day').unix()
```

Reply:
Thank you.  The following modification worked:

        const todayEpoch = date.startOf('day').unix()
        const nextDayEpoch = date.add(1, 'day').startOf('day').unix()

Once I create the test.js, I'll open a separate pull quest to add it to the repo.